<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteDialog</class>
 <widget class="QDialog" name="DeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Delete Interval</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="deleteIntervalMergeWithFollowingRadioButton">
     <property name="text">
      <string>Merge with following interval (remove the beat at the end of interval)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="deleteIntervalMergeWithPreceedingRadioButton">
     <property name="text">
      <string>Merge with preceeding interval (remove the beat at the beginning of interval)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="deleteIntervalMergePreceedingWithFollowingRadioButton">
     <property name="text">
      <string>Merge preceeding interval with following interval (remove both ends of interval)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="deleteIntervalMoveLaterIntervalsRadioButton">
     <property name="text">
      <string>Move later beats to fill the gap (all beats to the right of the selected beat will be moved left)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="deleteIntervalMoveEarlierIntervalsRadioButton">
     <property name="text">
      <string>Move earlier beats to fill the gap  (all beats to the left of the selected beat will be moved right)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeleteDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeleteDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
